redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,345 @@
+            displayName: CodeReady Workspaces version
+            path: cheVersion
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:label'
+          - description: The current status of the application
+            displayName: Status
+            path: cheClusterRunning
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes.phase'
+  keywords:
+    - codeready
+    - workspaces
+    - devtools
+    - developer
+    - ide
+    - java
+  displayName: CodeReady Workspaces
+  provider:
+    name: Red Hat, Inc.
+    url: https://developers.redhat.com/products/codeready-workspaces/overview/
+  maturity: stable
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  version: 1.1.0
+  maintainers:
+    - email: eivantsov@redhat.com
+      name: Eugene Ivantsov
+  description: >
+    A collaborative Kubernetes-native development solution that delivers OpenShift workspaces and in-browser IDE for rapid cloud application development.
+    This operator installs PostgreSQL, Red Hat SSO, and the CodeReady Workspaces server, as well as configures all three services.
+
+    ## Pre-Reqs
+
+    The operator service account requires cluster-admin privileges if you enable either `server.selfSignedCerts` or `auth.openShiftoAuth`.
+    After the operator is installed, grant the **codeready-operator** service account such privileges:
+
+
+    ```
+    oc adm policy add-cluster-role-to-user cluster-admin -z codeready-operator -n=$CodeReadyNamespace
+    ```
+
+    ## How to Install
+
+    Press the **Install** button, choose the upgrade strategy, and wait for the **Installed** Operator status.
+
+    When the operator is installed, create a new CR of Kind CheCluster (click the **Create New** button).
+    The CR spec contains all defaults (see below).
+
+    You can start using CodeReady Workspaces when the CR status is set to **Available**, and you see a URL to CodeReady Workspaces.
+
+    ## Defaults
+
+    By default, the operator deploys CodeReady Workspaces with:
+
+    * Bundled PostgreSQL and Red Hat SSO
+
+    * Common PVC strategy
+
+    * Auto-generated passwords
+
+    * HTTP mode (non-secure routes)
+
+    * Regular login (no login with OpenShift)
+
+    ## Installation Options
+
+    CodeReady Workspaces operator installation options include:
+
+    * Connection to external database and Keycloak/Red Hat SSO
+
+    * Configuration of default passwords and object names
+
+    * TLS mode
+
+    * PVC strategy (once shared PVC for all workspaces, PVC per workspace, or PVC per volume)
+
+    * Authentication options
+
+    ### External Database and Keycloak
+
+    To instruct the operator to skip deploying PostgreSQL and Red Hat SSO and connect to an existing DB and Red Hat SSO or Keycloak instead:
```

Comment:
Is this also what someone would need to do to get SSO with their OpenShift credentials? If so please make that explicit since this is a frequently asked question by our field teams.

Reply:
No. It's about using an existing RH SSO instance rather than deploying a new one. Same with database. 

Feature you are talking about is OpenShift oAuth which goes through Keycloak anyway. The question is whether or not you need Keycloak or you have an existing one.

Reply:
Thanks for clarifying Eugene.